webapi/src/main/scala/org/knora/webapi/routing/admin/UsersRouteADM.scala

```diff
@@ -592,7 +416,7 @@ final case class UsersRouteADM(
    */
   private def addUserToProjectAdminMembership(): Route =
     path(usersBasePath / "iri" / Segment / "project-admin-memberships" / Segment) { (userIri, projectIri) =>
-      post { requestContext =>
+      post { ctx =>
```

Comment:
it's called `requestContext` everywhere else, so it would probably be more consistent to leave it at that here too

Reply:
Actually AFAIS `requestContext` has been replaced everywhere by `ctx`, so at least it's consistent ;)